content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx

```diff
@@ -6,141 +6,166 @@ metaDescription: 'Learn how to deploy a TypeScript application to Deno Deploy.'
-```
+### Steps
 
-In the generated `prisma/schema.prisma`, add the following schema:
+1. Open your terminal and navigate to a location of your choice.
+2. Run the following commands to set up your application.
 
-```prisma
-generator client {
-  provider        = "prisma-client-js"
-  previewFeatures = ["deno"]
-  output          = "../generated/client"
-}
+  ```terminal
+  mkdir prisma-deno-deploy
+  cd prisma-deno-deploy
+  deno run -A --unstable npm:prisma init
+  ```
 
-datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
-}
+3. In the generated `prisma/schema.prisma`, add the following schema:
 
-model Log {
-  id      Int    @id @default(autoincrement())
-  level   Level
-  message String
-  meta    Json
-}
+  ```prisma
+  generator client {
+    provider        = "prisma-client-js"
+    previewFeatures = ["deno"]
+    output          = "../generated/client"
+  }
 
-enum Level {
-  Info
-  Warn
-  Error
-}
-```
+  datasource db {
+    provider = "postgresql"
+    url      = env("DATABASE_URL")
+  }
+
+  model Log {
+    id      Int    @id @default(autoincrement())
+    level   Level
+    message String
+    meta    Json
+  }
+
+  enum Level {
+    Info
+    Warn
+    Error
+  }
+  ```
 
-This data model will store incoming requests from your application.
+  This data model defines the `Log` model. Later in the guide, you create an application that uses the `Log` model to store data for incoming requests from the application.
 
-Note that this already includes the preview feature flag `deno` to active the current preview support for Deno in Prisma Client, and that it defines a custom `output` location for your Prisma Client. This is also required for Deno applications.
+  Note that the `schema.prisma` file includes the preview feature flag `deno` to activate the current preview support for Deno in Prisma Client, and it also defines a custom `output` location for your Prisma Client. This is a requirement for Deno applications.
 
-In your `.env` file, replace the current placeholder connection string `postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public` with your actual PostgreSQL connection string.
+4. In your `.env` file, replace the current placeholder connection string `postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public` with your PostgreSQL connection string.
 
 ## 2. Create the database schema
 
-With the data model created and your database connection configured, you can now create the data model in your database:
+With the data model in place and your database connection configured, you can now apply the data model to your database.
+
+### Steps
 
 ```terminal
 deno run -A --unstable npm:prisma db push
 ```
 
-This command installs Prisma Client in your project. There is no `package.json` file yet, so the command also creates this file.
+### Result
+
+With this command, you achieve 2 results. 
+
+* install Prisma Client in your project
+* create `package.json` file
```

Comment:
This is only a side effect of the command (as it seems to run `prisma generate` under the hood). The actual result achieved is that the models are created in your previously empty database.

Reply:
I've rewritten this section. Can you check again?

Reply:
Yep that works.